CHANGELOG.md

```diff
@@ -16,7 +24,6 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Marius Rackwitz](https://github.com/mrackwitz)
   [#4137](https://github.com/CocoaPods/CocoaPods/pull/4137)
 
-
```

Comment:
Please don't remove this, this is an intentional double new line between releases.